packages/expo-modules-autolinking/scripts/ios/xcode_env_generator.rb

```diff
@@ -1,6 +1,24 @@
 require 'open3'
 require 'pathname'
 
+def generate_or_remove_xcode_env_updates_file!()
+  project_directory = Pod::Config.instance.project_root
+  xcode_env_file = File.join(project_directory, '.xcode.env.updates')
+
+  ex_updates_native_debug = ENV['EX_UPDATES_NATIVE_DEBUG'] == '1'
```

Comment:
Might want to make this accept "true" as well, or perhaps check for any value that's not "", "0", or "false".